druntime/src/core/demangle.d

```diff
@@ -2901,3 +2911,63 @@ private string toStringConsume (immutable ManglingFlagInfo[] infos, ref ushort b
     }
     return null;
 }
+
+const CXX_DEMANGLER _no_cxx_demangler = cast(CXX_DEMANGLER) -1;
```

Comment:
Instead of using a sentinel value, would it be possible to implement a `noCXXDemangler` function that just returns its argument, or something of that spirit ? It would then allow you to branch in client code, and you could just call the function pointer unconditionally, either calling the real demangler or the shim that follows the same API.

Reply:
done